package.json

```diff
@@ -23,6 +23,7 @@
     "nextjs-redirect": "^5.2.1",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "react-gtm-module": "^2.0.11",
```

Comment:
[Package on NPM](https://www.npmjs.com/package/react-gtm-module), and [GitHub repository](https://github.com/alinemorelli/react-gtm)